juan-pablo-lopez/Tarea 1/index.html

```diff
@@ -14,51 +14,52 @@
     <title>Cover Magazine</title>
 </head>
 <body>
-    <div class="grid-container">
-        <div class="grid-tittle">
+    <main class="grid-container">
+        <header class="grid-tittle">
             <h1>NOISE</h1>
-        </div>
-        <div class="grid-deck" >
+        </header>  
+        <section class="grid-deck" >
             <h2>FASHION AND TRAVEL MAGAZINE</h2>
-        </div>
-        <div class="grid-article" >
-            <div> 
+        </section>
+        <article class="grid-article" >
+            <section> 
                 <p>OVID</p> 
                 <p>MODITAE</p>
                 <p>CTOTATI </p>
                 <p>ONSEDIST</p> 
-                <p>LIQUUNT</p></div> 
-            </div>
-            <div class="grid-lead-article-line" > 
-                <div>
-                    <p>STYLE </p> 
-                    <p>ALITAE </p> 
-                    <p>MUS AUT</p>
-                    <p>Aliquod igmin cuptat</p>
-                </div>
-        </div>
-        <div class="grid-suporting-overlines" >
-                <div>
-                    <p> ANDAM </p>
-                    <p>DOLUPTA</p> 
-                    <p> SEDITA </p>  
-                    <p> IUNT</p>
-                    <br>
-                    <br>
-                    <p>CUENTEM</p>
-                    <p>VENDES LOAK </p> 
-                    <p> ANDAM OPIM </p>
-                </div>
-        </div>
-        <div class="grid-website-link" >
-            <img src="img/barcode.PNG" alt="barccode">
-        </div>
-        <div class="grid-footer">
-            <div>
+                <p>LIQUUNT</p>
+            </section> 
+        </article>
+        <article class="grid-lead-article-line" > 
+            <section>
+                <p>STYLE </p> 
```

Comment:
@juanpablo-lopez01 , the content for all this

does not add semantic Value, try to put all the text inside one

the play with the width and text-wrap properties in CSS